pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIsPipeline.changelog.md

```diff
@@ -0,0 +1,4 @@
+# 1.0.0
+2022-01-14 (Date of Last Commit)
+
+* Initial Release of BroadInternalRNAWithUMIs pipeline.
```

Comment:
Should we include a short description of the purpose of the pipeline? Something like "The BroadInternalRNAWithUMIs pipeline wraps the RNAWithUMIs pipeline and performs additional steps that rely on Broad specific infrastructure."

Reply:
Hmm.  Did that, but still failing in smart test.